srv/components/system/config/set_hostname.sls

```diff
@@ -15,9 +15,12 @@
 # please email opensource@seagate.com or cortx-questions@seagate.com.
 #
 
+Update hostname file permissions:
+  cmd.run:
+    - name: "chattr -i /etc/hostname"
```

Comment:
shouldn't we restore that protection once hostname is updated?

Reply:
Updated